instance/migrations/0040_auto_20160420_0754.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_instance_states(apps, schema_editor):
+    SingleVMOpenEdXInstance = apps.get_model("instance", "SingleVMOpenEdXInstance")
+    for instance in SingleVMOpenEdXInstance.objects.iterator():
+        server_status = instance.server_status
+        if server_status:
+            if server_status.state_id in ("new", "started", "active", "rebooting"):
+                instance._status_to_waiting_for_server()
+            elif server_status.state_id in ("booted", "ready"):
+                instance._status_to_waiting_for_server()
+                instance._status_to_configuring_server()
+                instance._status_to_running()
+            elif server_status.state_id == "provisioning":
+                instance._status_to_waiting_for_server()
+                instance._status_to_configuring_server()
+            elif server_status.state_id == "terminated":
+                instance._status_to_waiting_for_server()
+                instance._status_to_configuring_server()
+                instance._status_to_running()
+                instance._status_to_terminated()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0039_auto_20160419_0921'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='singlevmopenedxinstance',
+            name='status',
```

Comment:
Shouldn't this be `name='_status'` and shouldn't there be `db_column='status'` on the line below?